nav2_costmap_2d/include/nav2_costmap_2d/costmap_2d_publisher.hpp

```diff
@@ -77,11 +78,13 @@ class Costmap2DPublisher
   {
     costmap_pub_->on_activate();
     costmap_update_pub_->on_activate();
+    costmap_raw_pub_->on_activate();
```

Comment:
Reading the code, I'm not clear yet on what costmap_raw_pub_ is. A comment would be helpful. I suppose there is some kind of raw costmap that is being published?

Reply:
I wanted to change the name of all the pubs actually. The `costmap_raw_pub` publishes the costmap using the native values from the costmap2D array. The original `costmap_pub' is publishing a translated occupancy grid version for rviz visualization. Would it perhaps be more helpful to change the `costmap_pub` to `costmap_rviz_pub`?

Reply:
No, I wouldn't specify who I expect to be listening. Just keep it to a topic name that anyone can listen to.